<HR COLOR=blue>
日志时间：2017-01-20 09:32:19,158 [8] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:34:14,523 [11] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:34:42,371 [11] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:34:43,325 [3] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:34:44,047 [11] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:35:08,577 [11] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：请求内容:{
  "DealerId": "D2233",
  "FromTime": "2016-09-01 00:00:00.123",
  "EndTime": "2016-09-30 00:00:00.123",
  "TotalPoint": "5000",
  "TotalMoney": "500",
  "ISAgree": "Y",
  "IntegraList": [
    {
      "UserId": "sample string 1",
      "Phone": "sample string 2",
      "ConsumeType": "sample string 3",
      "PointCost": null,
      "TotalCost": 45,
      "ConsumePoints": 100,
      "RewardPoints": 20,
      "ConsumeDate": "2016-09-01 00:00:00.123",
     "Comment": "sample string 9",
      "IdentityNumber": "sample string 10",
      "VIN": "sample string 11"
    },
    {
      "UserId": "sample string 1",
      "Phone": "sample string 2",
      "ConsumeType": "sample string 3",
      "PointCost": 40,
      "TotalCost": 50,
      "ConsumePoints": 500,
      "RewardPoints": 70,
      "ConsumeDate": "2016-09-01 00:00:00.123",
     "Comment": "sample string 9",
      "IdentityNumber": "sample string 10",
      "VIN": "sample string 11"
    }
  ]
} <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 09:35:08,577 [11] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：请求内容:{
  "DealerId": "D2233",
  "FromTime": "2016-09-01 00:00:00.123",
  "EndTime": "2016-09-30 00:00:00.123",
  "TotalPoint": "5000",
  "TotalMoney": "500",
  "ISAgree": "Y",
  "IntegraList": [
    {
      "UserId": "sample string 1",
      "Phone": "sample string 2",
      "ConsumeType": "sample string 3",
      "PointCost": null,
      "TotalCost": 45,
      "ConsumePoints": 100,
      "RewardPoints": 20,
      "ConsumeDate": "2016-09-01 00:00:00.123",
     "Comment": "sample string 9",
      "IdentityNumber": "sample string 10",
      "VIN": "sample string 11"
    },
    {
      "UserId": "sample string 1",
      "Phone": "sample string 2",
      "ConsumeType": "sample string 3",
      "PointCost": 40,
      "TotalCost": 50,
      "ConsumePoints": 500,
      "RewardPoints": 70,
      "ConsumeDate": "2016-09-01 00:00:00.123",
     "Comment": "sample string 9",
      "IdentityNumber": "sample string 10",
      "VIN": "sample string 11"
    }
  ]
} <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 09:35:18,649 [11] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：调用GetDMSIntegralList接口积分结算过程中，DMS的积分清单（成功），方法：GetDMSIntegralList 传入参数：{"DealerId":"D2233","FromTime":"\/Date(1472659200123)\/","EndTime":"\/Date(1475164800123)\/","TotalPoint":5000,"TotalMoney":500,"ISAgree":"Y","IntegraList":[{"UserId":"sample string 1","Phone":"sample string 2","ConsumeType":"sample string 3","PointCost":0,"TotalCost":45,"ConsumePoints":100,"RewardPoints":20,"ConsumeDate":"\/Date(1472659200123)\/","Comment":"sample string 9","IdentityNumber":"sample string 10","VIN":"sample string 11"},{"UserId":"sample string 1","Phone":"sample string 2","ConsumeType":"sample string 3","PointCost":40,"TotalCost":50,"ConsumePoints":500,"RewardPoints":70,"ConsumeDate":"\/Date(1472659200123)\/","Comment":"sample string 9","IdentityNumber":"sample string 10","VIN":"sample string 11"}]}||时间：2017/1/20 9:35:18 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 09:35:18,649 [11] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：调用GetDMSIntegralList接口积分结算过程中，DMS的积分清单（成功），方法：GetDMSIntegralList 传入参数：{"DealerId":"D2233","FromTime":"\/Date(1472659200123)\/","EndTime":"\/Date(1475164800123)\/","TotalPoint":5000,"TotalMoney":500,"ISAgree":"Y","IntegraList":[{"UserId":"sample string 1","Phone":"sample string 2","ConsumeType":"sample string 3","PointCost":0,"TotalCost":45,"ConsumePoints":100,"RewardPoints":20,"ConsumeDate":"\/Date(1472659200123)\/","Comment":"sample string 9","IdentityNumber":"sample string 10","VIN":"sample string 11"},{"UserId":"sample string 1","Phone":"sample string 2","ConsumeType":"sample string 3","PointCost":40,"TotalCost":50,"ConsumePoints":500,"RewardPoints":70,"ConsumeDate":"\/Date(1472659200123)\/","Comment":"sample string 9","IdentityNumber":"sample string 10","VIN":"sample string 11"}]}||时间：2017/1/20 9:35:18 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 09:35:42,942 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.InvalidOperationException: 在 HttpRequest.GetBufferedInputStream 的调用方填充内部存储之前访问了 BinaryRead、表单、文件或 InputStream。
   在 System.Web.HttpRequest.GetEntireRawContent()
   在 System.Web.HttpRequest.get_InputStream()
   在 Vcyber.BLMS.WebApi.Controllers.SettlementController.GetDMSIntegralList(DMSIntergralListV data) 位置 D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\Controllers\SettlementController.cs:行号 90
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:37:42,831 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.InvalidOperationException: 在 HttpRequest.GetBufferedInputStream 的调用方填充内部存储之前访问了 BinaryRead、表单、文件或 InputStream。
   在 System.Web.HttpRequest.GetEntireRawContent()
   在 System.Web.HttpRequest.get_InputStream()
   在 Vcyber.BLMS.WebApi.Controllers.SettlementController.GetDMSIntegralList(DMSIntergralListV data) 位置 D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\Controllers\SettlementController.cs:行号 90
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:39:33,683 [11] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:39:35,783 [11] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:39:38,951 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:39:43,165 [11] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:39:45,664 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:39:49,088 [11] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:49:47,632 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:50:05,007 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:50:26,306 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:50:28,955 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:50:39,112 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:50:40,657 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:50:41,423 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:50:42,654 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:51:22,767 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:51:33,733 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:51:34,888 [29] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:51:36,109 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:51:37,433 [31] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 09:52:03,712 [19] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：请求内容: <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 09:52:03,712 [19] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：请求内容: <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 09:52:43,541 [19] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.InvalidOperationException: 在 HttpRequest.GetBufferedInputStream 的调用方填充内部存储之前访问了 BinaryRead、表单、文件或 InputStream。
   在 System.Web.HttpRequest.GetEntireRawContent()
   在 System.Web.HttpRequest.get_InputStream()
   在 Vcyber.BLMS.WebApi.Controllers.SettlementController.GetDMSIntegralList(DMSIntergralListV data) 位置 D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\Controllers\SettlementController.cs:行号 90
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 10:03:50,195 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：请求内容: <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 10:03:50,195 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：请求内容: <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 10:03:51,459 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：请求内容:a <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 10:03:51,459 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：请求内容:a <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 10:04:52,639 [9] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.InvalidOperationException: 在 HttpRequest.GetBufferedInputStream 的调用方填充内部存储之前访问了 BinaryRead、表单、文件或 InputStream。
   在 System.Web.HttpRequest.GetEntireRawContent()
   在 System.Web.HttpRequest.get_InputStream()
   在 Vcyber.BLMS.WebApi.Controllers.SettlementController.GetDMSIntegralList(DMSIntergralListV data) 位置 D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\Controllers\SettlementController.cs:行号 90
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 10:05:00,815 [10] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 10:05:03,303 [9] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 10:05:04,229 [9] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 13:29:02,668 [11] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：新增积分消费接口请求内容: {
  "DealerId": "D1109",
  "ConsumeType": "6",
  "TotalCost": "500",
  "ConsumePoints": "1200",
  "ConsumeDate": "20160928183622",
  "PhoneNumber": "13522258920",
  "Operator": "",
  "OperatorDate": "20160928183622",
  "IdentityNumber": "210504199005038714",
  "VIN": "12345656543456734"
   } <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 13:29:02,668 [11] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：新增积分消费接口请求内容: {
  "DealerId": "D1109",
  "ConsumeType": "6",
  "TotalCost": "500",
  "ConsumePoints": "1200",
  "ConsumeDate": "20160928183622",
  "PhoneNumber": "13522258920",
  "Operator": "",
  "OperatorDate": "20160928183622",
  "IdentityNumber": "210504199005038714",
  "VIN": "12345656543456734"
   } <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 14:05:26,410 [9] <BR>
日志级别：ERROR     <BR>
日志类：bm-web-api-controller [(null)] <BR>
信息： <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.WebApi\App_Start\FilterConfig.cs <BR>
 <HR Size=1>System.Web.Http.HttpResponseException: Processing of the HTTP request resulted in an exception. Please see the HTTP response returned by the 'Response' property of this exception for details.
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger, CancellationToken cancellationToken)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.ReadContentAsync(HttpRequestMessage request, Type type, IEnumerable`1 formatters, IFormatterLogger formatterLogger)
   在 System.Web.Http.ModelBinding.FormatterParameterBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.HttpActionBinding.<ExecuteBindingAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
<HR COLOR=blue>
日志时间：2017-01-20 14:05:39,930 [8] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：积分结算接口请求内容:{
              "IntegraList":[{
                      "UserId":null
                      ,"Phone":"18701467845"
                      ,"IdentityNumber":"110103198705291829"
                      ,"VIN":"LBETLBFC6FY021024"
                      ,"ConsumeType":"0"
                      ,"ConsumePoints":1250
                      ,"TotalCost":90
                      ,"PointCost":0
                      ,"RewardPoints":12
                      ,"ConsumeDate":"2014-04-22 00:00:00.123"
                      ,"Comment":""
                     }
                     ,{
                      "UserId":null
                      ,"Phone":"18701467845"
                      ,"IdentityNumber":"110103198705291829"
                      ,"VIN":"LBETLBFC6FY021024"
                      ,"ConsumeType":"0"
                      ,"ConsumePoints":1250
                      ,"TotalCost":90
                      ,"PointCost":0
                      ,"RewardPoints":12
                      ,"ConsumeDate":"2014-04-22 00:00:00.123"
                      ,"Comment":""
                     }]
              ,"FromTime":"2016-04-01 00:00:00.123"
              ,"ISAgree":"Y"
              ,"EndTime":"2016-04-30 00:00:00.123"
              ,"DealerId":"D0103"
              ,"TotalMoney":"2500"
              ,"TotalPoint":"0"
         } <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 14:05:39,930 [8] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：积分结算接口请求内容:{
              "IntegraList":[{
                      "UserId":null
                      ,"Phone":"18701467845"
                      ,"IdentityNumber":"110103198705291829"
                      ,"VIN":"LBETLBFC6FY021024"
                      ,"ConsumeType":"0"
                      ,"ConsumePoints":1250
                      ,"TotalCost":90
                      ,"PointCost":0
                      ,"RewardPoints":12
                      ,"ConsumeDate":"2014-04-22 00:00:00.123"
                      ,"Comment":""
                     }
                     ,{
                      "UserId":null
                      ,"Phone":"18701467845"
                      ,"IdentityNumber":"110103198705291829"
                      ,"VIN":"LBETLBFC6FY021024"
                      ,"ConsumeType":"0"
                      ,"ConsumePoints":1250
                      ,"TotalCost":90
                      ,"PointCost":0
                      ,"RewardPoints":12
                      ,"ConsumeDate":"2014-04-22 00:00:00.123"
                      ,"Comment":""
                     }]
              ,"FromTime":"2016-04-01 00:00:00.123"
              ,"ISAgree":"Y"
              ,"EndTime":"2016-04-30 00:00:00.123"
              ,"DealerId":"D0103"
              ,"TotalMoney":"2500"
              ,"TotalPoint":"0"
         } <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 14:06:06,695 [8] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：调用GetDMSIntegralList接口积分结算过程中，DMS的积分清单（成功），方法：GetDMSIntegralList 传入参数：{"DealerId":"D0103","FromTime":"\/Date(1459440000123)\/","EndTime":"\/Date(1461945600123)\/","TotalPoint":0,"TotalMoney":2500,"ISAgree":"Y","IntegraList":[{"UserId":null,"Phone":"18701467845","ConsumeType":"0","PointCost":0,"TotalCost":90,"ConsumePoints":1250,"RewardPoints":12,"ConsumeDate":"\/Date(1398096000123)\/","Comment":"","IdentityNumber":"110103198705291829","VIN":"LBETLBFC6FY021024"},{"UserId":null,"Phone":"18701467845","ConsumeType":"0","PointCost":0,"TotalCost":90,"ConsumePoints":1250,"RewardPoints":12,"ConsumeDate":"\/Date(1398096000123)\/","Comment":"","IdentityNumber":"110103198705291829","VIN":"LBETLBFC6FY021024"}]}||时间：2017/1/20 14:06:06 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 14:06:06,695 [8] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：调用GetDMSIntegralList接口积分结算过程中，DMS的积分清单（成功），方法：GetDMSIntegralList 传入参数：{"DealerId":"D0103","FromTime":"\/Date(1459440000123)\/","EndTime":"\/Date(1461945600123)\/","TotalPoint":0,"TotalMoney":2500,"ISAgree":"Y","IntegraList":[{"UserId":null,"Phone":"18701467845","ConsumeType":"0","PointCost":0,"TotalCost":90,"ConsumePoints":1250,"RewardPoints":12,"ConsumeDate":"\/Date(1398096000123)\/","Comment":"","IdentityNumber":"110103198705291829","VIN":"LBETLBFC6FY021024"},{"UserId":null,"Phone":"18701467845","ConsumeType":"0","PointCost":0,"TotalCost":90,"ConsumePoints":1250,"RewardPoints":12,"ConsumeDate":"\/Date(1398096000123)\/","Comment":"","IdentityNumber":"110103198705291829","VIN":"LBETLBFC6FY021024"}]}||时间：2017/1/20 14:06:06 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:47:49,307 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================1 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:47:49,307 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================1 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:46,751 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================1 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:46,751 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================1 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:49,818 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：调用GetDMSIntegralList接口积分结算传入参数：D01032016/4/30 0:00:002016/4/1 0:00:00Y25000 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:49,818 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：调用GetDMSIntegralList接口积分结算传入参数：D01032016/4/30 0:00:002016/4/1 0:00:00Y25000 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,356 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================2 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,356 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================2 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,386 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================3 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,386 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================3 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,662 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================7 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,662 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================7 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,668 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================8 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,668 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================8 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,805 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================9 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,805 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================9 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,807 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================10 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,807 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================10 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,810 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================4 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,810 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================4 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,813 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================12 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,813 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================12 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,814 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================13 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,814 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================13 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,815 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================14 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,815 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================14 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,847 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================15 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,847 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================15 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,849 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================13 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,849 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================13 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,850 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================14 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,850 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================14 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,853 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================15 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,853 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================15 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,854 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================5 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,854 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================5 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,854 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================6 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,854 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：=============================================6 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,855 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：调用GetDMSIntegralList接口积分结算（成功），方法：GetDMSIntegralList 传入参数：{"DealerId":"D0103","FromTime":"\/Date(1459440000123)\/","EndTime":"\/Date(1461945600123)\/","TotalPoint":0,"TotalMoney":2500,"ISAgree":"Y","IntegraList":[{"UserId":null,"Phone":"18701467845","ConsumeType":"0","PointCost":0,"TotalCost":90,"ConsumePoints":1250,"RewardPoints":12,"ConsumeDate":"\/Date(1398096000123)\/","Comment":"","IdentityNumber":"110103198705291829","VIN":"LBETLBFC6FY021024"},{"UserId":null,"Phone":"18701467845","ConsumeType":"0","PointCost":0,"TotalCost":90,"ConsumePoints":1250,"RewardPoints":12,"ConsumeDate":"\/Date(1398096000123)\/","Comment":"","IdentityNumber":"110103198705291829","VIN":"LBETLBFC6FY021024"}]}||时间：2017/1/20 17:49:52 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1><HR COLOR=blue>
日志时间：2017-01-20 17:49:52,855 [9] <BR>
日志级别：INFO      <BR>
日志类：loginfo [(null)] <BR>
信息：调用GetDMSIntegralList接口积分结算（成功），方法：GetDMSIntegralList 传入参数：{"DealerId":"D0103","FromTime":"\/Date(1459440000123)\/","EndTime":"\/Date(1461945600123)\/","TotalPoint":0,"TotalMoney":2500,"ISAgree":"Y","IntegraList":[{"UserId":null,"Phone":"18701467845","ConsumeType":"0","PointCost":0,"TotalCost":90,"ConsumePoints":1250,"RewardPoints":12,"ConsumeDate":"\/Date(1398096000123)\/","Comment":"","IdentityNumber":"110103198705291829","VIN":"LBETLBFC6FY021024"},{"UserId":null,"Phone":"18701467845","ConsumeType":"0","PointCost":0,"TotalCost":90,"ConsumePoints":1250,"RewardPoints":12,"ConsumeDate":"\/Date(1398096000123)\/","Comment":"","IdentityNumber":"110103198705291829","VIN":"LBETLBFC6FY021024"}]}||时间：2017/1/20 17:49:52 <BR>
所在文件：D:\project\Vcyber.BLMS\Vcyber.BLMS.Common\LogService.cs <BR>
 <HR Size=1>